core/domain/translation_fetchers_test.py

```diff
@@ -37,12 +38,22 @@ def test_get_translation_from_model(self):
         model_instance = translation_models.MachineTranslatedTextModel.get(
             model_id)
         self.assertEqual(
-            translation_domain.MachineTranslatedText(
-                model_instance.source_language_code,
-                model_instance.target_language_code,
-                model_instance.source_text,
-                model_instance.translated_text
-            ).to_dict(),
+            translation_fetchers.get_translation_from_model(
+                model_instance).to_dict(),
             translation_domain.MachineTranslatedText(
                 'en', 'es', 'hello world', 'hola mundo').to_dict()
         )
+
+    def test_get_translation_for_text_with_no_translation_returns_none(self):
+        translation = translation_fetchers.get_translation_for_text(
+            'en', 'es', 'untranslated_text')
+        self.assertIsNone(translation)
+
+    def test_get_translation_for_text_with_existing_translation(
```

Comment:
Can you add the expected outcome in the test name?

Reply:
Added.